static/styles/user_circles.css

```diff
@@ -6,8 +6,6 @@
     border-radius: 50%;
     border: 1px solid;
     float: left;
-    position: relative;
-    top: 5px;
 }
```

Comment:
How does this commit work for other places we show user presence circles, e.g. in typeahead or the left sidebar?  We might need to do a bit more broad work.